user-metrics/mdservice/spcs.go

```diff
@@ -0,0 +1,386 @@
+// spcs.go
+
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	sf "github.com/snowflakedb/gosnowflake"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"log"
+	"net"
+	"net/http"
+	"os"
+	"strings"
+)
+
+const (
+	ConfigFile          = "config.yaml"
+	SpcsPluginName      = "spcs"
+	SpcsTokenFilePath   = "/snowflake/session/token"
+	ComputePoolQuery    = "SHOW COMPUTE POOLS;"
+	RoleQuery           = "SELECT CURRENT_ROLE();"
+	NameColumn          = "name"
+	StateColumn         = "state"
+	OwnerColumn         = "owner"
+	StateActiveValue    = "active"
+	DnsPrefix           = "metrics"
+	DnsSuffix           = "snowflakecomputing.internal"
+	MetricsPort         = "9001"
+	MetaSpcsPrefix      = SpcsPluginName + "_"
+	MetaComputePoolName = "__meta_" + MetaSpcsPrefix + "compute_pool_name"
```

Comment:
simplify pls to a single const string. You don't use `MetaSpcsPrefix` and `SpcsPluginName` anywhere but here